src/tools/rados/rados.cc

```diff
@@ -429,6 +557,13 @@ static int do_put(IoCtx& io_ctx, RadosStriper& striper,
       }
       continue;
     }
+
+#ifndef WITH_LIBRADOSSTRIPER
```

Comment:
This `#ifndef` appears to be a noop because it's inside `#else // WITH_LIBRADOSSTRIPER`